### DISCUSSION TITLE
How to add a new page in Elsa Studio

### DISCUSSION THREAD

**Discussion by quytrieu37** [elsa-workflows/elsa-studio#279] [Q&A]
Hi everyone,
I’m trying to add a new page in Elsa Studio by creating a new page in the /Pages folder with the following code:
```
@page "/testpage"
@layout MainLayout
<PageTitle>My New Page</PageTitle>
```
But when navigating to that endpoint, the browser displays a message Sorry, there's nothing at this address..
 I’m new to Blazor and Elsa Studio, any help or guidance would be greatly appreciated.
Thank you so much!

**Comment by geffzhang** [comment]
https://github.com/dupies/medupiramaboea-gmail.com
